packages/compass-serverstats/electron/main/index.js

```diff
@@ -1,42 +1,30 @@
 const electron = require('electron');
+const path = require('path');
 const app = electron.app;
 const BrowserWindow = electron.BrowserWindow;
-const watch = require('watch');
-const exec = require('child_process').exec;
-const path = require('path');
-
-const ROOT_DIR = path.resolve(`${__dirname}/../..`);
-const SRC_DIR = path.join(ROOT_DIR, 'src');
 
 let mainWindow;
 
 function createWindow() {
-  mainWindow = new BrowserWindow({ width: 1200, height: 800, webPreferences: {nodeIntegration: true }});
-  mainWindow.loadURL(`file://${__dirname}/../renderer/index.html`);
+  mainWindow = new BrowserWindow({
+    width: 1200,
+    height: 800,
+    webPreferences: { nodeIntegration: true }
+  });
+  mainWindow.loadURL(
+    process.env.COMPASS_INDEX_RENDERER_URL ||
+      `file://${path.resolve(__dirname, '..', 'index.html')}`
```

Comment:
I realize this is less bad than before, but at the same time, [`url.pathToFileURL`](https://nodejs.org/api/url.html#url_url_pathtofileurl_path) is a thing and takes care of some edge cases that this doesn’t (such as characters that are special in URLs, if they happen to show up in the file path)

Reply:
I totally didn't know about `pathToFileURL`, this makes a lot of sense and I'll fix it, ty!

Reply:
Done in 48440c3